scripts/docker/install_netcdf_cxx4.sh

```diff
@@ -147,6 +149,28 @@ function install() {
   cmake --install ${build_dir}
 }
 
+function get_shared_lib_extension() {
+  local os_type=$(uname)
+  case "${os_type}" in
+  Darwin)
+    echo "dylib"
+    ;;
+  Linux)
+    echo "so"
+    ;;
+  MINGW* | MSYS* | MYSYS*)
```

Comment:
are we going to use windows in github actions?

Reply:
Not right now because the build sever installation is used.  Locally, I run this in git bash. Running `uname` yields:
```
$ uname
MINGW64_NT-10.0-19045
```
That is covered in the case at line 161.